PiBridgeMaster.c

```diff
@@ -314,7 +314,7 @@ int PiBridgeMaster_Run(void)
 				piIoComm_writeSniff2A(enGpioValue_High, enGpioMode_Output);
 				piIoComm_writeSniff2B(enGpioValue_High, enGpioMode_Output);
 
-				usleep_range(9000, 9000);
+				usleep_range(8500, 8500);
```

Comment:
Looks good so far. But since we are at this code part, maybe add a comment that explains why we use the special value 8500? Maybe even use a define for the 8500 and add the comment there...

Reply:
I agree. The purpose of the delay is absolutely less than clear. A year from now nobody will remember what it's even for.

Reply:
full ack, will do